### PR TITLE
Adding support for installing multiple gcloud components

### PR DESCRIPTION
**Pull request by peterlin741**
**Summary**

- [discussion context](https://github.com/GoogleCloudPlatform/cloud-code-intellij-internal/pull/3615#discussion_r590688062)

Followed the docs to update the command to accept multiple components during install
- https://cloud.google.com/sdk/gcloud/reference/components/install
- e.g. `gcloud components install alpha beta --quiet`

**Ran the following:**
- `mvn fmt:format`
- `mvn clean install -DskipTests`
  - Could not get the tests to compile on a clean main branch, so skipped them

**Testing**
- Did not manually test yet

### CODE REVIEW

**Review by chanseokoh — COMMENTED** [review]
LGTM

Just make the tests pass. I ran unit tests on the main branch, and it's passing.

https://github.com/GoogleCloudPlatform/appengine-plugins-core/actions/runs/661440907

You might have mistaken error messages from some unit tests for actual errors?

File: src/main/java/com/google/cloud/tools/managedcloudsdk/components/SdkComponentInstaller.java [hunk 1]
```diff
@@ -58,18 +59,42 @@
...
+      List<SdkComponent> components,
+      ProgressListener progressListener,
+      ConsoleListener consoleListener)
+      throws InterruptedException, CommandExitException, CommandExecutionException {
+
+    String message;
+    if (components.size() == 1) {
+      message = "Installing " + components.get(0).toString();
+    } else {
+      message = "Installing components";
```
[the remark this answers is not in the document]
Comment (peterlin741) [reply src/main/java/com/google/cloud/tools/managedcloudsdk/components/SdkComponentInstaller.java]: good idea, updated

File: src/main/java/com/google/cloud/tools/managedcloudsdk/components/SdkComponentInstaller.java [hunk 2]
```diff
@@ -58,18 +60,45 @@
...
     List<String> command =
-        Arrays.asList(
-            gcloudPath.toString(), "components", "install", component.toString(), "--quiet");
+        new ArrayList<String>() {
+          {
+            add(gcloudPath.toString());
+            add("components");
+            add("install");
+          }
+        };
```
Comment (chanseokoh) [inline src/main/java/com/google/cloud/tools/managedcloudsdk/components/SdkComponentInstaller.java:98]: How about this?
```java
    List<String> command = new ArrayList<>();
    Collections.addAll(command, gcloudPath.toString(), "components", "install");
```
Or
```java
    List<String> command =
        new ArrayList<>(
            Arrays.asList(gcloudPath.toString(), "components", "install"));
```

**Review by chanseokoh — APPROVED** [review]

File: src/main/java/com/google/cloud/tools/managedcloudsdk/components/SdkComponentInstaller.java (hunk 2 above)
Comment (peterlin741) [reply src/main/java/com/google/cloud/tools/managedcloudsdk/components/SdkComponentInstaller.java]: Updated to use `Collections::addAll`

**Pull request merged**